kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/repository/AbstractRepository.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.good.patterns.challenges.repository;
+
+import com.kodilla.good.patterns.challenges.entity.EntityInterface;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+abstract public class AbstractRepository implements RepositoryInterface {
+    public String save(EntityInterface entity) {
+        if (!getList().contains(entity)) {
+            List<String> ids = getList().stream().map(entityElement -> entityElement.getId().orElse("")).collect(Collectors.toList());
```

Comment:
id powinno byc unikalne, zgadza sie? Dodatkowo sprawdziles linie wyzej ze entity istnieje w liscie. 

Jesli powyzsze sa prawdziwe, to wynik powinien byc pojedynczym stringiem, nie lista

Reply:
No nie Bardzo bo element nie ma identyfikatora i trzeba mu dopiero go nadać